var/spack/repos/builtin/packages/metis/package.py

```diff
@@ -33,6 +33,13 @@ class Metis(CMakePackage, MakefilePackage):
 
     depends_on("c", type="build")  # generated
 
+    variant(
+        "no_warning", 
+        default=False, 
+        description="Disable failed partition warning print on all ranks",
+    )
+    patch("no_warning.patch", when="@5:+no_warning")
+
```

Comment:
```suggestion
    variant(
        "no_warning",
        default=False,
        description="Disable failed partition warning print on all ranks",
    )
    patch("no_warning.patch", when="@5:+no_warning")

```

this fixes black checking.